package.json

```diff
@@ -35,7 +35,7 @@
   ],
   "license": "MIT",
   "engines": {
-    "node": ">=4.2.0"
+    "node": ">=6"
```

Comment:
I looked through the news, and LTS starts at 6.9.0. https://nodejs.org/en/blog/release/v6.9.0/

Reply:
We don't want to specify the LTS version as the one we require. I'm pretty sure we required 4.2.0 because it had a feature that we needed, didn't it?

It feels silly to change our requirements based on LTS and not "what we require to run".

Reply:
Aren't we basing our support on LTS versions?

Reply:
I think we are saying that we will definitely support the latest LTS version, but why break earlier versions if we don't have a reason to? So, if there's a feature that 6 gives us that we want to use, then that's fine, but if we aren't using something that isn't in 4, there's no point in crashing out (regarding your comment about just exiting). And so, what's the point in saying we only support 6.9.0 if there's no functional difference that we don't use with 6.0.0? People should obviously make sure their version is up to date with 6.9.0, but ... it seems arbitrary to say "6.9.0" unless there's a feature in that that isn't in earlier 6 versions that we use.

Reply:
That's fair.

Reply:
Yeah, so the way I understand Node's LTSes, 6.0 to 6.8 were released during before they entered the LTS schedule, and 6.9+ were released during the LTS schedule. But overall, v6 is the LTS version.

But yeah, if The Lounge is compatible with 6.0 to 6.8, we might as well not require 6.9. I see at https://github.com/thelounge/lounge/pull/280 that we already had this discussion lol, but the key difference is that v4.2 was released 1 month after 4.0, while there is a 6-month gap between 6.0 and 6.9.
This kinda tells me we need an extra build for `6.0` (on top of `6`, the latest LTS), but meh.

Anyway, I think 6.0.0 is reasonable, and we can bump this if The Lounge is not compatible with this one.